bundle/config/generate/job.go

```diff
@@ -0,0 +1,34 @@
+package generate
+
+import (
+	"github.com/databricks/cli/libs/dyn"
+	"github.com/databricks/cli/libs/dyn/yamlsaver"
+	"github.com/databricks/databricks-sdk-go/service/jobs"
+)
+
+var jobOrder = yamlsaver.NewOrder([]string{"name", "new_cluster", "job_clusters", "existing_cluster_id", "compute", "tasks"})
```

Comment:
Just noticed: the top-level `new_cluster` and `existing_cluster_id` fields should never show up.

For multi-task jobs (the default), these are task-level fields.